File: 백준/Bronze/6825. Body Mass Index/Body Mass Index.py
import sys
input = sys.stdin.readline

w = float(input().rstrip())
h = float(input().rstrip())
r = w/(h*h)

if r > 25: print('Overweight')
elif 18.5 <= r <= 25: print('Normal weight')
else: print('Underweight')

File: 백준/Bronze/6825. Body Mass Index/Body Mass Index.cc
#include <stdio.h>

int main(void) {
    double weight, height;
    scanf("%lf", &weight);
    scanf("%lf", &height);

    double bmi = weight / (height * height);
    if (bmi > 25) printf("Overweight");
    else if (bmi < 18.5) printf("Underweight");
    else printf("Normal weight");
    
    return 0;
}